squad-server/index.js

```diff
@@ -265,6 +307,14 @@ export default class SquadServer extends EventEmitter {
         ...oldPlayerInfo[player.steamID],
         ...player
       }));
+
+      for (const player of this.players) {
+        if (typeof oldPlayerInfo[player.steamID] === 'undefined') continue;
+        if (player.teamID !== oldPlayerInfo[player.steamID].teamID)
+          this.emit('PLAYER_TEAM_CHANGE', player);
+        if (player.squadID !== oldPlayerInfo[player.steamID].squadID)
+          this.emit('PLAYER_SQUAD_CHANGE', player);
```

Comment:
Maybe we should pass their old team/squad, including if that was undefined. Perhaps in the format:
```
{
  player: { ... },
  old: x, // can be undefined if they just joined.
  new: y
}
```

Reply:
I believe I have done this the way you asked